src/operator/tensor/elemwise_binary_broadcast_op.h

```diff
@@ -155,6 +181,22 @@ struct binary_broadcast_kernel {
     }
   }
 };
+
+template<int req, typename OP>
+struct csr_dns_csr_broadcast_kernel {
+  template <typename DType, typename CType, typename RType>
+  MSHADOW_XINLINE static void Map(int row, const DType *csr_data, const CType *csr_indices,
+                                  const RType *csr_indptr, const DType *dns,
+                                  DType *out, const nnvm::dim_t row_length, bool col_vec) {
```

Comment:
col_vec could be part of the template

Reply:
This flag may actually be abandoned later, will do if it still exists later.